src/Workleap.OpenApi.MSBuild/msbuild/tools/Workleap.OpenApi.MSBuild.targets

```diff
@@ -2,6 +2,7 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <OpenApiEnabled Condition="'$(OpenApiEnabled)' == ''">true</OpenApiEnabled>
+    <SpectralProfile Condition="'$(SpectralProfile)' == ''">backend</SpectralProfile>
```

Comment:
Should namespace the property by OpenApi prefix

Reply:
Is it worth mentioning that under the hood it is using Spectral? Or only OpenApiProfile (B2B or B4F) could be good enough?

Reply:
As per offline discussion, this PR should only affect the PR execution spectral profile, but as suggested, I think there is value in renaming this to `OpenApiServiceProfile` in case we do want to do profile specific work in the future (saves a csproj property update for users)